source/codegen/validate_linux_rt.py

```diff
@@ -39,16 +47,25 @@ def _need_linux_rt_feed_update(metadata_dir: str, changed_files: Set[str]) -> bo
     codegen_changes = _get_codegen_changes(changed_files)
     _print_changed_files("Codegen changes not affecting RT:", codegen_changes)
     remaining_changes = changed_files - codegen_changes
-    test_changes = _get_test_changes(changed_files)
+    test_changes = _get_test_changes(remaining_changes)
     _print_changed_files("Test related changes not affecting RT:", test_changes)
     remaining_changes -= test_changes
+    import_changes = _get_import_changes(remaining_changes)
```

Comment:
While I was here, I noticed that we'd list markdown changes and import changes as likeley needing RT updated to pull them in but I  think we can treat them the same as metadata or test changes and ignore them for the evaluation purposes.